src/index.html

```diff
@@ -17,16 +17,37 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="index.html"
+      method="get"
+      class="search-bar search-bar--big"
+      data-qa="big"
+    >
+      <div class="search-bar__content-wrapper search-bar--big__content-wrapper">
+        <input
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+          class="search-bar__content-wrapper__input search-bar--big__content-wrapper__input"
+        />
+      </div>
+    </form>
+    <form
+      action="index.html"
+      method="get"
+      class="search-bar search-bar--small"
+      data-qa="small"
+    >
```

Comment:
Both forms have the same `action` attribute pointing to `index.html`. If these forms are intended to perform different actions, consider changing the `action` attribute to point to different URLs or endpoints.